vcsclient/logger.go

```diff
@@ -5,6 +5,7 @@ const (
 	successfulRepoExtraction = "Extracted repository successfully"
 	creatingPullRequest      = "Creating new pull request:"
 	fetchingOpenPullRequests = "Fetching open pull requests in"
+	fetchingPullRequestById  = "Fetching pull requests by id in "
```

Comment:
```suggestion
	fetchingPullRequestById  = "Fetching pull requests by id in"
```

Redundant space, our logger adds space automatically.